torch/csrc/jit/python/script_init.cpp

```diff
@@ -51,7 +51,7 @@ namespace jit {
 using ::c10::Argument;
 using ::c10::FunctionSchema;
 
-using ResolutionCallback = std::function<py::function(std::string)>;
+using ResolutionCallback = std::function<py::object(std::string)>;
```

Comment:
Do we also need to modify `torch/_C/__init__.pyi.in`?
```
# Defined in torch/csrc/jit/python/script_init.cpp
ResolutionCallback = Callable[[str], Callable[..., Any]]
```

Reply:
Good point, but let me do it in followup PR, because I would also like to add type annotations for https://github.com/pytorch/pytorch/blob/233e4ebdb60c8353dab765986d87a45e4213e640/torch/_jit_internal.py#L95